pkg/interface/package.json

```diff
@@ -13,6 +13,7 @@
     "@tlon/indigo-react": "^1.2.19",
     "@tlon/sigil-js": "^1.4.3",
     "@urbit/api": "file:../npm/api",
+    "@welldone-software/why-did-you-render": "^6.1.0",
```

Comment:
This should specifically be a dev dependency. This sort of utility seems like it shouldn't be in the bundle at all, in the same way I don't bundle VSCode with Landscape, but I don't think React has such a concept, so that's moot.